src/SignalRServiceExtension/Bindings/LiteServiceEndpoint.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Azure.SignalR;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    [JsonObject(NamingStrategyType = typeof(CamelCaseNamingStrategy))]
```

Comment:
Add summary for public class.

Reply:
a Azure SignalR Service endpoint? What do you mean by members is a subset? Regarding `ServiceEndpoint`? Better to use complete sentence and object reference if needed. And suggest to add some comments about properties are used to select for customized routing.